### ISSUE TITLE
Menu bar icon colors

### ISSUE THREAD

**Issue by cdbattags** [rokartur/Micmute#3]
Could we get an update to make the menu bar icon more in line with other defaults?

<img width="325" alt="Screenshot 2024-11-12 at 14 16 01" src="https://github.com/user-attachments/assets/4645ac1a-9f64-4951-b928-c71ac98d538a">

Thanks!

**Comment by cdbattags** [comment]
I believe the answer is something like:

`image?.isTemplate = true`

**Comment by rokartur** [comment]
I solved this problem, in the beta version, there is a stable, but currently you need to build yourself

**Comment by rokartur** [comment]
yes, here is the list of features I want to do ([dev branch](https://github.com/rokartur/Micmute/tree/dev)):

- [x] select input device
- [x] set volume after unmute
- [ ] silent mode (when system volume is 0 => mute micro)
- [x] check mute status
- [ ] push to talk mode
- [x] menubar icon behavior on left click (toggle mute or show menu)
- [ ] option to leave the notification on the screen until the microphone is unmuted or always displayed

Currently I don't have as much time as I would like. Studying and creating other larger projects are taking up more time these days
